### ISSUE TITLE
Having trouble running docker-compose

### ISSUE THREAD

**Issue by reilly3000** [dessalines/flowchat#155]
/~flowchat$ docker-compose up
ERROR: Service 'flowchat' failed to build: Error parsing reference: "node:9 as node-builder" is not a valid repository/tag: invalid reference format

I'm running:
Docker version 1.13.1, build 092cba3
docker-compose version 1.20.1, build 5d8c71b
on ubuntu 17.10

I tried a few different base image tags with no luck. I also tried changing or eliminating the alias, but that caused other issues.

**Comment by dessalines** [comment]
I think you're on a really old version of docker, here's how to upgrade it for ubuntu: 
https://askubuntu.com/questions/472412/how-do-i-upgrade-docker#473720

for the record i'm on `Docker version 18.03.0-ce`. 

Docker compose version looks fine.

**Comment by reilly3000** [comment]
Silly mistake, thank you for your help. I was indeed running a super old version of docker obtained with using `sudo apt install docker` - I needed to clean out that install and get `sudo apt install docker-ce` after adding docker repos. The system stood itself up fine after that and works great! I did to run docker-compose up twice as the first time the pg db had a connection issue. The second time worked flawlessly.
